eng/scripts/Language-Settings.ps1

```diff
@@ -119,28 +121,21 @@ function Get-AllPackageInfoFromRepo ([string] $ServiceDirectory) {
 }
 
 function Get-rust-PackageInfoFromPackageFile([IO.FileInfo]$pkg, [string]$workingDirectory) {
-  #$pkg will be a FileInfo object for the cargo-metadata.json file in a package artifact directory
-  $package = Get-Content -Path $pkg.FullName -Raw | ConvertFrom-Json
-  $packageName = $package.name
-  $packageVersion = $package.vers
+  InstallAndImport-ModuleIfNotInstalled "PSToml" "0.3.1"
 
-  $crateFile = Get-ChildItem $pkg.DirectoryName -Filter '*.crate'
+  #$pkg will be a FileInfo object for the Cargo.toml file in a package artifact directory
+  $toml = Get-Content -Path $pkg.FullName -Raw | ConvertFrom-Toml
```

Comment:
Why not just use `cargo read-manifest` which JSONifies the TOML?

Reply:
that is much better